commit 069bfeff40cbd608e403efb5cd03102d73a85623 (refs/changes/38/5638/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-26T17:18:36+00:00 (7 months ago)
    
    TRITON-1141 convert sdc-agents-core to engbld framework (fix node version)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index fdd4711..1cfc5c7 100644
--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ JSSTYLE_FLAGS    = -o indent=4,doxygen,unparenthesized-return=0
 NODE_PREBUILT_VERSION=v0.12.14
 NODE_PREBUILT_TAG=gz
 ifeq ($(shell uname -s),SunOS)
-    NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
+    NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
 
 ENGBLD_REQUIRE := $(shell git submodule update --init deps/eng)
